### PR TITLE
FXIOS-281: Added Pocket event count

### PR DESCRIPTION
**Pull request by nbhasin2**
Counts the number of times a user opens pocket article from Firefox home pocket feed

### CODE REVIEW

**Review by dnarcese — CHANGES_REQUESTED** [review]

**Comment by nbhasin2** [comment]
# Request for data collection review form

**All questions are mandatory. You must receive review from a data steward peer on your responses to these questions before shipping new data collection.**

1) What questions will you answer with this data?

Counts the number of times a user opens pocket article from Firefox home pocket feed. 
This will let us know the click-through-rate (how many pocket articles are clicked on compared to new tab opens)

2) Why does Mozilla need to answer these questions?  Are there benefits for users? Do we need this information to address product or business requirements? 

It is to ensure we are giving users value on our new tab page & see the effect of changes in design

3) What alternative methods did you consider to answer these questions? Why were they not sufficient?

We had metrics in leanplum for this but nothing for our internal telemetry 

4) Can current instrumentation answer these questions?

Not really as this is a new event for our internal telemetry and the only event that existed prior to it is in Leanplum.

5) List all proposed measurements and indicate the category of data collection for each measurement, using the [Firefox data collection categories](https://wiki.mozilla.org/Firefox/Data_Collection) found on the Mozilla wiki.   

**Note that the data steward reviewing your request will characterize your data collection based on the highest (and most sensitive) category.**

<table>
  <tr>
    <td>Measurement Description</td>
    <td>Data Collection Category</td>
    <td>Tracking Bug #</td>
  </tr>
  <tr>
    <td>Internal telemetry</td>
    <td>Category 2</td>
    <td>https://github.com/mozilla-mobile/firefox-ios/issues/6195</td>
  </tr>
</table>

6) Please provide a link to the documentation for this data collection which describes the ultimate data set in a public, complete, and accurate way.

https://github.com/mozilla-mobile/firefox-ios/blob/main/Docs/metrics.md

7) How long will this data be collected?  Choose one of the following:
Until: 2021-10-01

8) What populations will you measure?

All users

9) If this data collection is default on, what is the opt-out mechanism for users?

In Firefox iOS there is a preference for telemetry labeled "Send Usage Data" that will be used as the opt-out mechanism.

10) Please provide a general description of how you will analyze this data.

We will look at how many times user taps on the pocket feed article that will let us know the click-through-rate (how many pocket articles are clicked on compared to new tab opens)

11) Where do you intend to share the results of your analysis?

Mozilla internal channels with our PM 

12) Is there a third-party tool (i.e. not Telemetry) that you are proposing to use for this data collection?

No 3rd party tool

**Comment by dnarcese** [comment]
r+ with one questions - where did 2021-10-01 come from? I think it should be 6 months.

# Data Review Form

1) Is there or will there be **documentation** that describes the schema for the ultimate data set in a public, complete, and accurate way? 

Yes, telemetry is documented in `docs/metrics.md`

2) Is there a control mechanism that allows the user to turn the data collection on and off? 

Yes, Firefox iOS provides data controls in the app settings

3) If the request is for permanent data collection, is there someone who will monitor the data over time?

This should be re-reviewed w/ @athomasmoz every 6mo

4) Using the **[category system of data types](https://wiki.mozilla.org/Firefox/Data_Collection)** on the Mozilla wiki, what collection type of data do the requested measurements fall under?

Type 2

5) Is the data collection request for default-on or default-off?

Default on

6) Does the instrumentation include the addition of **any *new* identifiers** (whether anonymous or otherwise; e.g., username, random IDs, etc.  See the appendix for more details)?

No

7) Is the data collection covered by the existing Firefox privacy notice? 

Yes

8) Does there need to be a check-in in the future to determine whether to renew the data? (Yes/No) (If yes, set a todo reminder or file a bug if appropriate)**

Should review all probes for necessity every 6mo w/ @athomasmoz 

9) Does the data collection use a third-party collection tool? 

No

File: Client/metrics.yaml [hunk 1]
```diff
@@ -836,3 +836,18 @@ experiments:
...
+pocket:
+  open_story:
+    type: counter
+    description: |
+        Counts the number of times a user opens
+        Pocket article from Firefox home Pocket feed
+    bugs:
+      - https://github.com/mozilla-mobile/firefox-ios/issues/6195
+    data_reviews:
+      - https://github.com/mozilla-mobile/firefox-ios/issues/6195
```
Comment (dnarcese) [inline Client/metrics.yaml:850]: We can also update this link to the PR now

**Review by dnarcese — APPROVED** [review]

**Comment by nbhasin2** [comment]
@dnarcese updated date to 6 months from today

**Comment by nbhasin2** [comment]
Merging this in it was a green build and last commit was only to update a data review link and doesn't affect the actual workings on the PR

**Pull request merged**